packages/graphiql-react/src/ui/dropdown.css

```diff
@@ -8,9 +8,11 @@
   padding: var(--px-4);
   font-family: var(--font-family);
   color: hsl(var(--color-neutral));
+  max-height: calc(var(--radix-dropdown-menu-content-available-height) - 10px);
```

Comment:
https://www.radix-ui.com/docs/primitives/components/dropdown-menu#constrain-the-contentsub-content-size

Reply:
Using all the available height seems too much for me, having the dropdown fill the whole height of the page looks weird. I'd propose to go with a smaller value here like `400px`.

Reply:
hardcoding 400px would provoke issues on small `vh`, we should use css' `min` function with it

https://github.com/graphql/graphiql/assets/7361780/703a8a3f-d227-475b-884a-e6d89bdf2cd7